flask_security/core.py

```diff
@@ -461,30 +461,48 @@ class Security(object):
 
     :param app: The application.
     :param datastore: An instance of a user datastore.
+    :param register_blueprint: to register the Security blueprint or not.
+    :param login_form: set form for the login view
+    :param register_form: set form for the register view
+    :param confirm_register_form: set form for the confirm register view
+    :param forgot_password_form: set form for the forgot password view
+    :param reset_password_form: set form for the reset password view
+    :param change_password_form: set form for the change password view
+    :param send_confirmation_form: set form for the send confirmation view
+    :param passwordless_login_form: set form for the passwordless login view
+    :param anonymous_user: class to use for anonymous user
     """
 
     def __init__(self, app=None, datastore=None, **kwargs):
         self.app = app
         self.datastore = datastore
-
+        self.kwargs = dict(register_blueprint=True,
+                           login_form=None,
```

Comment:
do you need to set all the default values to `None`?

Reply:
If I'm reading the code right, I think so, because the keys need to be set somewhere so that `_SecurityState` can initialize its attributes: https://github.com/mattupstate/flask-security/blob/develop/flask_security/core.py#L410 Previously that happened in the call to `_get_state`, but now that the kwargs can be passed to either `__init__` and/or `init_app`, this seemed to me the most DRY way of doing it. Does that make sense, or perhaps I am missing something?

Reply:
Then I think we should move the default values to `_get_state` function.